3.10/deployment-cluster-manual-start.md

```diff
@@ -193,6 +202,9 @@ arangod --server.endpoint tcp://0.0.0.0:8531 \
   --agency.activate true \
   --agency.size 3 \
   --agency.supervision true \
+  --agency.endpoint tcp://192.168.1.1:8531 \
+  --agency.endpoint tcp://192.168.1.2:8531 \
+  --agency.endpoint tcp://192.168.1.3:8531 \
```

Comment:
AFAIK only the first agent would have to be given on the command line.

Reply:
Alright cool, I'll update this now